app/forms/induction_choice_form.rb

```diff
@@ -38,7 +38,7 @@ def programme_choices(i18n_scope: "schools.induction_choice_form.options")
   end
 
   def opt_out_choice_selected?
-    programme_choice&.in? %i[school_funded_fip design_our_own no_early_career_teachers]
+    programme_choice&.in? %i[school_funded_fip design_our_own]
```

Comment:
Why this change? While this may make it work, the method is now semantically wrong